bitmap-team-awesome/lib/color-transform.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+const fs = require('fs');
+
+module.exports = exports = {};
+
+exports.changeColorBlue = function(buffer, bufferObject) {
+  var x;
+  var blueColorArray = buffer.slice(54, bufferObject.colorOffSet);
+  for (x=0;x<blueColorArray.length;x+=4){
```

Comment:
@meeksfred Your buffer offset, loop, and color offset all look good - additionally, the color value assignment looks good.  That said, this method could be refactored to handle each r,g,b color change as opposed to having 3 separate methods for this.  If you take a high level approach, it's easy to see that the interior components of this method are very similar to that of your `changeColorGreen` and `changeColorRed` methods.  This method could be refactored to accept a color string and then do the logic based on that - for example (**note** - this has not been tested and should be viewed as an example, not an implementation):

```
exports.changeColor = function(buffer, bufferObject, color) {
  var x;
  var colorArray = buffer.slice(54, bufferObject.colorOffSet);
  for (x = 0; x < colorArray.length; x+=4) {
    if (blue) // do transform stuff
    if (red) // do transform stuff
    if (green) // do transform stuff
  }
  fs.writeFile(`${__dirname}/../../assets/bitmaps/${color}.bmp`, buffer, function(err) {
    if (err) throw err;
    console.log(`new ${color} bitmap created`);
  });
}
```

Also note that the color has been passed into the writeFile path, thus allowing for better reusability/modularity.